src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
+        ball.setColor(new ColorSupplier().getRandomColor());
```

Comment:
Move ColorSupplier to a class field
```suggestion
        ball.setColor(colorSupplier.getRandomColor());
```